content/pages/docs/install.md

```diff
@@ -9,7 +9,7 @@ save_as: docs/install/index.html
 
 We recommend installing IPFS from a prebuilt package. You can obtain those from [dist.ipfs.io](https://dist.ipfs.io/#go-ipfs).
 
-For convenience here are the latest versions linked directly
+For convenience here are the latest versions linked directly:
 
 - <i class="fa fa-apple"></i> [Mac OS X 64bit](https://dist.ipfs.io/go-ipfs/v0.4.5/go-ipfs_v0.4.5_darwin-amd64.tar.gz)
```

Comment:
As unrelated note, we need to stop linking to it. I think it would be best to just link to dist.ipfs.io/#go-ipfs